zebedee/client_test.go

```diff
@@ -607,3 +608,95 @@ func TestClient_PublishedDataEndpoint(t *testing.T) {
 		})
 	})
 }
+
+func TestClient_PublishedIndexEndpoint(t *testing.T) {
+	ctx := context.Background()
+	path := "/publishedindex"
+
+	Convey("given a 200 response", t, func() {
+		documentContent, err := ioutil.ReadFile("./response_mocks/publishedindex.json")
+		So(err, ShouldBeNil)
+		body := httpmocks.NewReadCloserMock(documentContent, nil)
+		response := httpmocks.NewResponseMock(body, http.StatusOK)
+
+		httpClient := newMockHTTPClient(response, nil)
+		zebedeeClient := newZebedeeClient(httpClient)
+
+		Convey("when zebedeeClient.GetPublishedIndex is called", func() {
+			m, err := zebedeeClient.GetPublishedIndex(ctx, nil)
+
+			Convey("then the expected content is returned", func() {
+				So(err, ShouldBeNil)
+				So(m.Count, ShouldEqual, 3)
+				So(m.Items, ShouldNotBeEmpty)
+				So(m.Items, ShouldHaveLength, 3)
+				mItem := m.Items[0]
+				So(mItem.URI, ShouldResemble, "/releases")
+				So(m.Limit, ShouldEqual, 3)
+				So(m.Offset, ShouldEqual, 0)
```

Comment:
This will pass even if offset is not specified in the Zebedee response (0 is the default value). Is that the intention?

Reply:
TBH the paging variables are only there for future compatibility should we implement it. So no need to worry about it right now.